org.elysium.test/src/org/elysium/test/regression/GrammarRegressions.xtend

```diff
@@ -25,6 +25,25 @@ class GrammarRegressions {
 	extension ValidationTestHelper validator
 
 	@Test
+	def void schemeAliasDefine() throws Exception {
+		val model='''
+			#(define anAlias something)
+			refersTo = \anAlias
+		'''.parse
+
+		model.assertNoErrors
+
+		val define=model.expressions.head
+		Assert.assertTrue(define instanceof Assignment)
+
+		val referencingAssignment=model.expressions.last
+		Assert.assertTrue(referencingAssignment instanceof Assignment)
+		val ref=(referencingAssignment as Assignment).value
+		Assert.assertTrue(ref.toString,ref instanceof Reference)
+		Assert.assertEquals("anAlias", (ref as Reference).assignment.name)
```

Comment:
Could you extract the alias name? I know abstraction should be limited in tests, but simple concatenation of string constants isn't too much indirection (especially with Xtend's template expressions).